keyboards/keyap/info.json

```
@@ -0,0 +1,45 @@
+{
+    "manufacturer": "PenEngineering",
+    "keyboard_name": "Keyapad",
+    "maintainer": "PenEngineering",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
```

As per the Imgur link in the readme, this very clearly uses a development board style microcontroller.
```suggestion
    "development_board": "elite_c",
```